alfasim_sdk/variables.py

```diff
@@ -41,6 +45,7 @@ class Scope(Enum):
 class SecondaryVariable:
     name: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
     caption: str = attrib(validator=[instance_of(str), check_string_is_not_empty])
+    type = attrib(validator=instance_of(Type), default=Type.Double)
```

Comment:
Also, I think that you need to define a new name for this attribute, otherwise you will override the builtin [type](https://docs.python.org/3/library/functions.html#type) =)